app/index.js

```diff
@@ -62,6 +62,8 @@ module.exports = function App(config) {
   // Attach RouteManager to app object, the primary set of mockyeah API methods.
   app.routeManager = new RouteManager(app);
 
+  app.proxying = app.config.proxy;
```

Comment:
You should set the configuration default for `proxy` on https://github.com/mockyeah/mockyeah/pull/126/files#diff-2018087f584c4398b5c3a23fc0e5f9dbR24

IMO, it is best in the situation to be explicit rather than depending on the falsiness of `undefined`.

Reply:
@ryanricard Added.